contracts/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "contracts"
+version = "0.1.0"
+authors = ["Nicholas Rodrigues Lordello <nicholas.lordello@gnosis.pm>"]
+edition = "2018"
+
+[[bin]]
+name = "vendor"
+required-features = ["bin"]
+
+[features]
+default = []
+bin = [
+    "anyhow",
+    "env_logger",
+    "ethcontract-generate",
+    "log",
+    "serde_json",
+]
+
+[dependencies]
+ethcontract = { version = "0.9",  default-features = false }
+serde = "1.0"
+
+# [bin-dependencies]
```

Comment:
Should this be uncommented?

Reply:
Its not an official `Cargo.toml` as far as I'm aware. I *think* its just for delimiting the dependencies that are only added when enabling the `bin` feature for building the `vendor` script.

Reply:
This crate has a lib target and a bin target. Cargo currently does not support specifying dependencies for those individually so the workaround is to make the dependencies optional and collect them in a feature.